GUI/Main.Designer.cs

```diff
@@ -124,7 +124,6 @@ private void InitializeComponent()
             this.splitContainer1.SuspendLayout();
             ((System.ComponentModel.ISupportInitialize)(this.ModList)).BeginInit();
             this.ModListContextMenuStrip.SuspendLayout();
-            this.StatusPanel.SuspendLayout();
```

Comment:
Add `this.statusStrip1.SuspendLayout();`

Reply:
Can't:

```
  Main.Designer.cs(127,31): error CS1061: 'StatusStrip' does not contain a definition for 'SuspsendLayout' and no extension method 'Suspse
ndLayout' accepting a first argument of type 'StatusStrip' could be found (are you missing a using directive or an assembly reference?)
```

Reply:
Done